client/grpc/grpc.go

```diff
@@ -73,7 +74,7 @@ func unaryInterceptor(target string) grpc.UnaryClientInterceptor {
 		carrier := headersCarrier{Ctx: ctx}
 		err := span.Tracer().Inject(span.Context(), opentracing.TextMap, &carrier)
 		if err != nil {
-			log.FromContext(ctx).Errorf("failed to inject tracing headers: %v", err)
+			log.FromContext(ctx).Error("failed to inject tracing headers", slog.Any("error", err))
```

Comment:
Do we still need the log package here for the ctx?

Reply:
yes, unfortunately, `slog` does not have a feature to support logging propagation via context...
This allows us to create loggers with extra data attributes per request...